test/k8sT/Services.go

```diff
@@ -831,6 +835,61 @@ var _ = Describe("K8sServicesTest", func() {
 			wg.Wait()
 		}
 
+		// This function tests NodePort srvices using IPV6 addresses
+		// It is the job of the caller to make sure that all the node have assigned
+		// routable IPV6 addresses reachable from other nodes.
+		testNodePortIPv6 := func(testFromOutside bool, data *v1.Service) {
+			var wg sync.WaitGroup
+
+			k8s1IPv6 := getIPv6AddrForIface(k8s1NodeName, helpers.SecondaryIface)
+			k8s2IPv6 := getIPv6AddrForIface(k8s2NodeName, helpers.SecondaryIface)
+
+			Expect(k8s1IPv6).ToNot(Equal(""), "Cannot get IPv6 address for K8s1 node")
+			Expect(k8s2IPv6).ToNot(Equal(""), "Cannot get IPv6 address for K8s2 node")
```

Comment:
Because the expectation is inside a function, we need to use `ExpectWithOffset`. Also, I think we can use `ToNot(BeEmpty()` or `ToNot(HaveLen(0)` (whichever exists :) )